chia/full_node/mempool_manager.py

```diff
@@ -215,7 +215,7 @@ async def pre_validate_spendbundle(self, new_spend: SpendBundle) -> NPCResult:
         """
         start_time = time.time()
         cached_result_bytes = await asyncio.get_running_loop().run_in_executor(
-            self.pool, get_npc_multiprocess, bytes(new_spend), self.constants.MAX_BLOCK_COST_CLVM
+            self.pool, get_npc_multiprocess, bytes(new_spend), self.limit_factor * self.constants.MAX_BLOCK_COST_CLVM
```

Comment:
part of me think we should even divide this by 3, since we're not deducting the cost of the program *size* nor the cost of created *coins*.